themes/default/content/blog/enhanced-search-intuitive-navigation-new-docs-experience/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Enhanced search & Navigation: The new Pulumi Docs experience"
+date: 2023-06-12
+meta_desc:  At Pulumi, we believe in exceptional documentation experiences that help people using Pulumi find what they need quickly and use it successfully.
+meta_image: meta.png
+authors:
+    - susan-evans
+    - anita-trimbur
+    - sean-holung
+tags:
+    - pulumi-releases
+    - features
+    - ux
+---
+
+Engineers spend a lot of time searching documentation for answers. At Pulumi, we believe in exceptional documentation experiences that help people using Pulumi find what they need quickly and use it successfully. Today, we announce an improved Pulumi documentation experience dedicated to all developers.
+<!--more-->
+
+We prioritized search and navigation, so you can easily browse and search for precisely what you want. Enjoy faster and more accurate searches, with lovely user experience touches to guide you to the right place. We also planned for the future, using design patterns that will grow with content and concepts.
```

Comment:
```suggestion
The improvements we've introduced to the Pulumi documentation experience focus on three areas - search, navigation and discoverability.  Together, they help every user of the Pulumi documentation find the answers they are looking for quickly, so that they can solve their infrastructure as code use case and leverage Pulumi effectively in their projects. These changes also introduce new design patterns and user experience elements that are being used across various previously independent parts of the Pulumi documentation experience, bringing these together as part of a more cohesive developer experience.
```